GOVERNANCE.md

```diff
@@ -0,0 +1,135 @@
+### io.js Build Team Working Group
+
+The io.js build team is jointly governed by a Working Group (WG)
+which is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+[README.md](./README.md#current-project-team-members).
+
+### Collaborators
+
+The [iojs/build](https://github.com/iojs/build),
+[iojs/docker-iojs](https://github.com/iojs/docker-iojs), and the
+[iojs/build-containers](https://github.com/iojs/build-containers)
+GitHub repositories are maintained by the WG and additional Collaborators
+who are added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These
+individuals are identified by the WG and their addition as
+Collaborators is discussed during the weekly WG meeting.
```

Comment:
This was brought up in the meeting. Need to establish a cadence for the WG meetings. Meeting on an "as needed" basis was proposed but concerns about nothing happening as a result were raised.

I like the idea of monthly meetings guaranteed with "as needed" meetings in between. Makes sure things get done, but that we aren't meeting just to meet.